### PR TITLE
chore: December Quiet Month Blog Post

### PR DESCRIPTION
**Pull request by groundwater**

### CODE REVIEW

File: blog/dec-quiet-month-21.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: A Quiet Place (Dec'21)
+date: 2021-09-27T00:00:00.000Z
+authors:
+  name: groundwater
+  url: 'https://github.com/groundwater'
+  image_url: 'https://github.com/groundwater.png?size=96'
+slug: a-quiet-place-21
+---
+ElectronJS will pause for the month of December 2021, but will return to full speed in January 2022.
```
Comment (erickzhao) [inline blog/dec-quiet-month-21.md:10]: We typically use `Electron` (no `js` suffix) in official communications, I think.

File: blog/dec-quiet-month-21.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: A Quiet Place (Dec'21)
+date: 2021-09-27T00:00:00.000Z
```
Comment (erickzhao) [inline blog/dec-quiet-month-21.md:3]: TODO: edit for when this is scheduled for release?

File: blog/dec-quiet-month-21.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+authors:
+  name: groundwater
+  url: 'https://github.com/groundwater'
+  image_url: 'https://github.com/groundwater.png?size=96'
+slug: a-quiet-place-21
+---
+
+Electron will pause for the month of December 2021, but will return to full speed in January 2022.
```
Comment (ckerr) [inline blog/dec-quiet-month-21.md:11]:
```suggestion
The Electron project will pause for the month of December 2021, then return to full speed in January 2022.
```

Minor change in tone

File: blog/dec-quiet-month-21.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Why is this happening?
+
+In short, while maintainers are happy and engaged with the project, _THE WORLD IS TIRED_. 
+December is a quiet month for most companies, so we want to give our maintainers a chance to recharge. We encourage other projects to consider similar measures.
+
+## Should I be worried about the future of Electron?
+
+No. We are able to take this step because the project is in good shape.
+Everyone is looking forward to 2022, and we expect good things to come!
```
Comment (ckerr) [inline blog/dec-quiet-month-21.md:41]: This is a very nice way to close the post

File: blog/dec-quiet-month-21.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+<div style={{width:"100%",height:"0",paddingBottom:"42%",position:"relative"}}>
+  <iframe src="https://giphy.com/embed/4QEGwDMDbOYC2G7m9p" width="100%" height="100%" style={{position:"absolute"}} frameBorder="0" className="giphy-embed" allowFullScreen></iframe>
+</div>
+<p><a href="https://giphy.com/gifs/shh-john-krasinski-a-quiet-place-4QEGwDMDbOYC2G7m9p">via GIPHY</a></p>
+
+---
+
+## What will be the same in December
+
+1. Zero-day and other major security-related releases will be published as necessary. Security incidents should report via [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md).
```
Comment (ckerr) [inline blog/dec-quiet-month-21.md:22]:
```suggestion
1. Zero-day and other major security-related releases will be published as necessary. Security incidents should be reported via [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md).
```

**Review by erickzhao — APPROVED** [review]

**Pull request merged**